media/css/firefox/whatsnew/whatsnew-lite.scss

```diff
@@ -63,158 +63,161 @@
 //* -------------------------------------------------------------------------- */
 // Page content
 
-.lite-container {
-    display: flex;
-    justify-content: flex-end;
-    align-items: center;
-    flex-direction: column;
-    margin-top: $spacing-md;
+.lite-body {
+    background-color: #89eae3;
+    background-image: url('/media/img/firefox/whatsnew/lite/background.jpg');
+    background-position: top right;
+    background-size: auto 100%;
+    background-repeat: no-repeat;
+    color: $color-gray-80;
+    position: relative;
+    @include font-firefox;
+    @media #{$mq-md} {
+        background-size: 150% auto;
+        background-position: top left;
+    }
+}
 
+.lite-content {
+    max-width: 1600px;
+    padding-left: $spacing-lg;
+    padding-right: $spacing-lg;
+    background-repeat: no-repeat;
+    background-position: top left;
+    background-size: 42%;
     @media #{$mq-md} {
-        min-height: 180px;
+        display: flex;
+        justify-content: flex-end;
+        background-image: url('/media/img/firefox/whatsnew/lite/photo.webp');
+        padding-bottom: $layout-xl;
     }
+}
 
-    @media #{$mq-lg} {
-        align-items: flex-start;
+.lite-copy {
+    @include text-body-md;
+    @media #{$mq-md} {
+        width: 58%;
     }
 }
 
+
 .lite-heading {
     display: flex;
     align-items: center;
+    justify-content: center;
+    margin-top: $spacing-xl;
+    margin-bottom: $spacing-xl;
 
     h1 {
-        @include font-firefox;
-
-        @include text-display-md;
-
         font-weight: normal;
-        text-align: start;
+        font-size: 2rem;
```

Comment:
Please try and avoid setting arbitrary font sizes, and choose from one of the predefined options in Protocol: https://protocol.mozilla.org/fundamentals/typography.html